string_challenges.py

```diff
@@ -1,28 +1,30 @@
 # Вывести последнюю букву в слове
 word = 'Архангельск'
-# ???
+print(word[-1])
 
 
 # Вывести количество букв "а" в слове
 word = 'Архангельск'
-# ???
+print(len([l for l in word if l == 'а']))
+print(len([l for l in word.lower() if l == 'а']))  # если без учета регистра
 
 
 # Вывести количество гласных букв в слове
 word = 'Архангельск'
-# ???
-
+print(len([l for l in word.lower() if l in ['а', 'е', 'и', 'о', 'у', 'э', 'ю', 'я']]))
```

Comment:
```suggestion
print(len([l for l in word.lower() if l in 'аеиоуэюя']))
```
Легче напечатать строку на клавиатуре, чем лист 😎
Популярный прием в олимпиадном программировании.